sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/ResourceActionCancelEventData.java

```diff
@@ -236,134 +234,117 @@ public ResourceActionCancelEventData setStatus(String status) {
      *
      * @param authorization the authorization value to set.
      * @return the ResourceActionCancelEventData object itself.
-     * @deprecated This method is no longer supported since v4.9.0.
-     * <p> Use {@link ResourceActionCancelEventData#setResourceAuthorization(ResourceAuthorization)} instead.
      */
-    @Deprecated
-    public ResourceActionCancelEventData setAuthorization(String authorization) {
-        try {
-            setResourceAuthorization(
-                DEFAULT_SERIALIZER_ADAPTER.deserialize(authorization, ResourceAuthorization.class,
-                    SerializerEncoding.JSON));
-        } catch (IOException ex) {
-            throw LOGGER.logExceptionAsError(new UncheckedIOException(ex));
-        }
+    public ResourceActionCancelEventData setResourceAuthorization(ResourceAuthorization authorization) {
+        this.authorization = authorization;
         return this;
     }
 
     /**
-     * Get the authorization property: The requested authorization for the operation.
+     * Get the claims property: The properties of the claims.
      *
-     * @return the authorization value.
+     * @return the claims value.
      */
-    public ResourceAuthorization getResourceAuthorization() {
-        return this.authorization;
+    public Map<String, String> getResourceClaims() {
+        return this.claims;
     }
 
     /**
-     * Set the authorization property: The requested authorization for the operation.
+     * Set the claims property: The properties of the claims.
      *
-     * @param authorization the authorization value to set.
+     * @param claims the claims value to set.
      * @return the ResourceActionCancelEventData object itself.
      */
-    public ResourceActionCancelEventData setResourceAuthorization(ResourceAuthorization authorization) {
-        this.authorization = authorization;
+    public ResourceActionCancelEventData setResourceClaims(Map<String, String> claims) {
+        this.claims = claims;
         return this;
     }
 
     /**
-     * Get the claims property: The properties of the claims.
+     * Get the correlationId property: An operation ID used for troubleshooting.
      *
-     * @return the claims value.
-     * @deprecated This method is no longer supported since v4.9.0.
-     * <p> Use {@link ResourceActionCancelEventData#getResourceClaims()} instead.
+     * @return the correlationId value.
      */
-    @Deprecated
-    public String getClaims() {
-        final Map<String, String> resourceClaims = getResourceClaims();
-        if (!resourceClaims.isEmpty()) {
-            try {
-                return DEFAULT_SERIALIZER_ADAPTER.serialize(resourceClaims, SerializerEncoding.JSON);
-            } catch (IOException ex) {
-                throw LOGGER.logExceptionAsError(new UncheckedIOException(ex));
-            }
-        }
-        return null;
+    public String getCorrelationId() {
```

Comment:
The diff for all the Resource* types is a little messy and hard to notice if there's actually any breaking change here. It might be easier to review this on API view. It just looks like reordering of methods and API view would confirm that.